### PR TITLE
Update naming history in README

### PR DESCRIPTION
**Pull request by ThomasHoooo**

### CODE REVIEW

**Comment by rexong** [comment]
Close this PR and reopen again.

**Comment by ThomasHoooo** [comment]
> Close this PR and reopen again.

Don't have to, just need to re-run. Now all the checks pass.

File: README.md [hunk 1]
```diff
@@ -14,6 +14,6 @@ optimized for use via a Command Line Interface (CLI), with all the benefits of a
 * The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
   * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
   * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
+* It is named `MyInsuRec` because it helps financial advisors to keep track of his client and meeting records.
```
Comment (rexong) [inline README.md:17]: should change to `clients and meetings`.

**Review by rexong — CHANGES_REQUESTED** [review]

**Review by rexong — APPROVED** [review]
LGTM

**Review by sikai00 — APPROVED** [review]
lgtm

**Pull request merged**